phoenix-core/src/main/java/org/apache/phoenix/coprocessor/MetaDataEndpointImpl.java

```diff
@@ -3295,11 +3295,14 @@ public void getVersion(RpcController controller, GetVersionRequest request, RpcC
                             PhoenixDatabaseMetaData.SYSTEM_CATALOG_TABLE_BYTES, HConstants.LATEST_TIMESTAMP, null,
                             request.getClientVersion());
         } catch (Throwable t) {
-            LOGGER.error("loading system catalog table inside getVersion failed", t);
-            ProtobufUtil.setControllerException(controller,
-                    ServerUtil.createIOException(
-                            SchemaUtil.getPhysicalTableName(PhoenixDatabaseMetaData.SYSTEM_CATALOG_NAME_BYTES,
-                                    isTablesMappingEnabled).toString(), t));
+            if (request.getClientVersion() >= MIN_SPLITTABLE_SYSTEM_CATALOG) {
+                LOGGER.error("loading system catalog table inside getVersion failed", t);
+                ProtobufUtil.setControllerException(controller,
+                        ServerUtil.createIOException(
+                                SchemaUtil.getPhysicalTableName(PhoenixDatabaseMetaData.SYSTEM_CATALOG_NAME_BYTES,
+                                        isTablesMappingEnabled).toString(), t));
+            }
```

Comment:
For else part, would it be better to just provide INFO log with client version and exact Exception that we are ignoring?

Reply:
this will result in so many logging because all older clients (pre4.15) will come to the else part.